DOL.WHD.Section14c.Web/src/modules/components/sectionEmployer/sectionEmployerTemplate.html

```diff
@@ -288,7 +288,6 @@
       <fieldset>
         <legend class="hide">Fiscal quarter end date</legend>
         <date-field date-val="formData.employer.fiscalQuarterEndDate"></date-field>
-        <label class="example">Example: 04 30 2016</label>
```

Comment:
I'm curious why this example text is gone from the content.  Should it still be somewhere?

Reply:
Please see description in the Pull Request. :)  I moved the Example text for date fields to the Date Field template so I could correlate the month/day/year fields with the individual month/day/year, and generate the proper IDs in case there are multiple date fields in a page. I realized this would cause duplicate Example text showing on pages I haven't touched yet. There were only two other date fields in the application that had the old Example text, so I went ahead and removed it from those pages.

Reply:
Oh, that makes sense. To make sure I understand, the example text will show up again in another PR?